### PR TITLE
code chellege

### PR DESCRIPTION
**Pull request by stacyburris**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: javascript/code-challenges/tree/__tests__/trees.test.js [hunk 1]
```diff
@@ -1,19 +1,34 @@
 'use strict';
```
Comment (adrianhuebner) [inline javascript/code-challenges/tree/__tests__/trees.test.js:2]: I would just like to see a test added for finding the max value in the tree
